src/ngraph/ngraph_emitter.cc

```diff
@@ -1000,18 +998,16 @@ void Emitter::CreateLayerOps() {
     // autodifferentiation (as with training).
     //----------------------------------------------------------------------------------------------
     if ((exe_mode_ == GraphExeMode::kTrain) && (use_global_stats)) {
-      // FIXME: We suspect there's a bug in the gradient calculations performed by this version of
-      // nGraph's BatchNorm operator. So for now we'll avoid using it.  -cconvey 2018-04-12.
+      // FIXME: We suspect there's a bug in the gradient calculations performed
+      // by this version of
+      // nGraph's BatchNorm operator. So for now we'll avoid using it.  -cconvey
+      // 2018-04-12.
```

Comment:
It would be great if we could nail down why we're getting these random reformatting changes.  Diff noise is annoying. But that's a battle for another day, another PR.